pythonforandroid/recipes/argon2-cffi/__init__.py

```diff
@@ -0,0 +1,17 @@
+from pythonforandroid.recipe import CompiledComponentsPythonRecipe
+
+
+class Argon2Recipe(CompiledComponentsPythonRecipe):
+    version = '20.1.0'
+    url = 'git+https://github.com/hynek/argon2-cffi'
```

Comment:
Non blocking suggestion: it's usually faster to download the archive directly rather than cloning
```suggestion
    url = 'https://github.com/hynek/argon2-cffi/archive/{version}.tar.gz'
```

Reply:
I'll try this is also, actually the problem we faced is that the libargon was not included in the zip, hence @obfusk suggested to use that as the url.

Reply:
Oh right! I missed that indeed sorry :+1: